test/helpers/cmd.go

```diff
@@ -485,6 +496,10 @@ func (res *CmdRes) WaitUntilMatchFilterLineTimeout(filter, expected string, time
 	body := func() bool {
 		lines, err := res.FilterLinesJSONPath(parsedFilter)
 		if err != nil {
+			if errors.Is(err, &JSONParseError{}) {
+				// We might have read a partial line; continue
+				return false
```

Comment:
Is there a way to emit a warning for that case? Not sure if it makes sense as probably nobody will look at passing test output :/.

Reply:
Looking at the uses of this function, we always wait for flows to appear, so it's safe to ignore parse errors.

I'd like to find a way to warn but I'm not sure the test library has that.